components/menteeDetails/MenteeDetails.js

```diff
@@ -60,72 +60,4 @@ class MenteeDetails extends Component {
   }
 }
 
-/*
```

Comment:
Is there a good way to capture in documentation what the blob looks like from the backend? It's nice to be able to see what the data blob looks like that this element is displaying, but perhaps that will get easier once Redux is in place and the data is coming from there instead of the API?

Reply:
usually what I think is best is to have a clean documentation from the backend/API side to show these kinds of blobs. Of course we can also have those setup in redux but this may also crowd the files

Reply:
Yeah I agree that it doesn't belong in the source here. I'll add some tasks for down the road to address the documentation.